rasa/nlu/test.py

```diff
@@ -784,10 +784,10 @@ def run_evaluation(
     data_path,
     model,
     report_folder=None,
-    successes_filename=None,
```

Comment:
these were ignored before, right? Could we also add this to the changelog / proposed changes?